lib/rails_response_dumper/colorize.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+COLORS = {
+  red: 31,
+  cyan: 36,
+  green: 32
+}.freeze
+
+COLORIZE = $stdout.tty?
+
+def print_color(text, color)
```

Comment:
This is all going into the global namesapce. I think we should wrap it in `module RailsResponseDumper`.